src/modules/auth/components/AuthModal/AuthModal.tsx

```diff
@@ -57,19 +57,36 @@ const AuthModal: FC<AuthModalProps> = (props) => {
 	};
 
 	const renderFooter = () => {
-		if (publicRuntimeConfig.ENABLE_LOGIN_FOOTER === 'true') {
-			return (
-				<div className="u-text-center u-bg-silver">
+		const showFooterLogin = publicRuntimeConfig.ENABLE_LOGIN_FOOTER === 'true';
```

Comment:
OPTIONAL: this environment variable was added so we could disable the footer when the 2 identify servers were merged on meemoo's side.
But we don't really need it anymore. So we can just enable the footer allways and delete the environment variable.